src/js/Aladin.js

```diff
@@ -1485,13 +1485,16 @@ export let Aladin = (function () {
     Aladin.prototype.removeLayers = Aladin.prototype.removeOverlays;
 
```

Comment:
I would add a jsdoc overlay type
```suggestion
/**
* @typedef {MOC|Catalog|ProgressiveCat|GraphicOverlay} Overlay
* @description Possible overlays
*/
```